SamplesDashboard/SamplesDashboard/SamplesDashboard.csproj

```diff
@@ -33,11 +33,13 @@
 
   <ItemGroup>
     <None Remove="ClientApp\src\components\Language.tsx" />
+    <None Remove="ClientApp\src\components\Samples.tsx" />
```

Comment:
the csproj file points to a location that seems not to exist. I am not sure whether this would break anything, but it's worth looking at it